.build/Makefile

```diff
@@ -0,0 +1,30 @@
+# set GOOARCH based on ARCH
+ifeq ($(ARCH),x86_64)
+GOARCH=amd64
+endif
+ifeq ($(ARCH),x86)
+GOARCH=386
+endif
+ifeq ($(GOARCH),)
+$(error "ARCH must be set to one of: x86, x86_64")
+endif
+
+# use "0.0.1" as default version if not supplied
```

Comment:
why `0.0.1` specifically? should it be some invalid version instead, e.g. `0.0.0`, or a git hash?
You can probably use something like `git describe --tags --dirty` for automatic versioning

Reply:
I still need to finalize how exactly versioning will work. Googet only supports versions using three dot notation, so it's a bit limited. This will become clearer once I get more understanding of how the package upload process works.